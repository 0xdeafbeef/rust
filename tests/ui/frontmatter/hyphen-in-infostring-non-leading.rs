--- Cargo-toml
//~^ ERROR: invalid infostring for frontmatter
---

// infostrings cannot have hyphens

#![feature(frontmatter)]

fn main() {}
