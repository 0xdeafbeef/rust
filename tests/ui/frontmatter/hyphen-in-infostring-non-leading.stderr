error: invalid infostring for frontmatter
  --> $DIR/hyphen-in-infostring-non-leading.rs:1:4
   |
LL | --- Cargo-toml
   |    ^^^^^^^^^^^
   |
   = note: frontmatter infostrings must be a single identifier immediately following the opening

error: aborting due to 1 previous error

